fulltext/__main__.py

```diff
@@ -12,17 +13,40 @@
 import fulltext
 
 
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
 def _handle_open(path):
     with open(path, 'rb') as f:
         return fulltext.get(f)
 
 
-def main(args=sys.argv[1:]):
+def test_backends():
```

Comment:
Can you rename this to `check_backends()`?

Also, please rename the CLI subcommand to `check` and mention it in `README`.